src/app/pages/LoginPage/loginPage.tsx

```diff
@@ -1,18 +1,26 @@
 import styles from "./loginPage.module.css";
+import Login from "../../components/Login/login";
 
 export default function LoginPage() {
   return (
     <div className={styles.container}>
-      <span className={styles.title}>STEINCHEN</span>
+      <div className={styles.title}>
+        <span className={styles.S}>S</span>
+        <span className={styles.T}>T</span>
+        <span className={styles.E}>E</span>
+        <span className={styles.I}>I</span>
+        <span className={styles.N}>N</span>
+        <span className={styles.C}>C</span>
+        <span className={styles.H}>H</span>
+        <span className={styles.E}>E</span>
+        <span className={styles.N}>N</span>
+      </div>
       <h3 className={styles.enterName}>Gib Deinen Namen ein</h3>
       <img
```

Comment:
I can't see this image nor the footer image in your deployment.

Reply:
But why? Whats wrong here?